Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM postgres:13
+ARG POSTGRES_DB
+ARG POSTGRES_USER
+ARG POSTGRES_PASSWORD
+
+ENV POSTGRES_DB=${POSTGRES_DB}
+ENV POSTGRES_USER=${POSTGRES_USER}
+ENV POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+
+ADD ./docker-entrypoint-initdb.d/init.sql /docker-entrypoint-initdb.d/
```

Comment:
Now that we removed the file, this command fails while trying to build the Dockerfile. We can remove it for now as we don't have any initialization script.
<img width="1384" alt="Screenshot 2023-07-11 at 12 32 37" src="https://github.com/JewelleryManagement/jewellery-management-service/assets/33765600/d732d942-e76d-460c-8b74-c5cc833614aa">

Reply:
Removed the line. Also added a Readme file